Adding Spanish Translation for the Pyladies Berlin Website

mjmolina opened the pull request:
Hello!

**Goal of this PR:** Website translation to Spanish.

fixes #87

mjmolina committed:
Adding Spanish Translation

sleepypioneer reviewed (approved):
Muy Bien, Gracias! 🙌 🇪🇸

cmaureir commented on content/pages/es/about.rst:
```diff
@@ -0,0 +1,24 @@
+
+Sobre PyLadies Berlin:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PyLadies es un grupo internacional de tutoría para géneros marginados, como personas no binarias, personas trans y mujeres en la tecnología, con el objetivo de ayudar a nuestra comunidad a convertirse en participantes activos y líderes en la comunidad de código abierto de Python. Todos los niveles son bienvenidos!
```
```suggestion
PyLadies es un grupo internacional de tutoría para géneros marginados, como personas no binarias, personas trans y mujeres en la tecnología, con el objetivo de ayudar a nuestra comunidad a convertirse en participantes activos y líderes en la comunidad de código abierto de Python. ¡Todos los niveles son bienvenidos!
```

cmaureir commented on content/pages/es/about.rst:
```diff
@@ -0,0 +1,24 @@
+
+Política de género:
+----------------------
+Creemos que el conocimiento es para todos. Al mismo tiempo, nuestros eventos tienen como objetivo principal empoderar a las mujeres y a las minorías de género en la comunidad tecnológica. Por ello, si no te identificas como parte de ninguna de ellas, te pedimos que seas consciente de esta situación y hagas que tu presencia sea discreta, por ejemplo, viniendo con una mujer o una persona de una min ...
```
```suggestion
Creemos que el conocimiento es para todas las personas. Al mismo tiempo, nuestros eventos tienen como objetivo principal empoderar a las mujeres y a las minorías de género en la comunidad tecnológica. Por ello, si no te identificas como parte de ninguna de ellas, te pedimos que seas consciente de esta situación y hagas que tu presencia sea discreta, por ejemplo, viniendo con una mujer o una persona de una minoría de género para garantizar el equilibrio de género, y que consideres que no se te escucha más ni ocupas más espacio que los demás en los debates y secciones de preguntas.
```

cmaureir commented on content/pages/es/about.rst:
```diff
@@ -0,0 +1,24 @@
+
+* Participa en nuestros eventos, asistiendo o como mentor `Meetup <https://www.meetup.com/de-DE/PyLadies-Berlin/>`__
+* Únete al espacio de trabajo de PyLadies `Slack <https://slackin.pyladies.com>`__ 
+* Pregúntanos!
```
```suggestion
* ¡Pregúntanos!
```

cmaureir commented on content/pages/es/jobs.rst:
```diff
@@ -0,0 +1,28 @@
+
+**¿Estás buscando un trabajo relacionado con Python?**
+
+Echa un vistazo a nuestro `slack`_ en el canal #jobs-europe canal y #jobs (este último canal tiene sobre todo Estados Unidos puestos de trabajo)
```
```suggestion
Echa un vistazo a nuestro `slack`_ en el canal #jobs-europe canal y #jobs (este último canal tiene puestos de trabajo en Estados Unidos)
```

cmaureir commented on content/pages/es/jobs.rst:
```diff
@@ -0,0 +1,28 @@
+
+**¿Tu empresa tiene buenos puestos en Python que quieras compartir?**
+
+  - Publica el trabajo directamente en nuestro `slack`_ en el canal #job-europe.
```
```suggestion
  - Publica el trabajo directamente en nuestro `slack`_ en el canal #jobs-europe.
```

cmaureir commented on content/pages/es/meetups.rst:
```diff
@@ -0,0 +1,40 @@
+:order: 2
+:date: 2024-05-21
+:icon: icon-speech-bubble
+:summary: Meetups mensual de Pyladies Berlin
```
```suggestion
:summary: Meetups mensual de PyLadies Berlin
```

cmaureir commented on content/pages/es/meetups.rst:
```diff
@@ -0,0 +1,40 @@
+temas relacionados con Python, como el desarrollo de software o la ciencia de datos.
+
+Además, cubrimos temas básicos en nuestros eventos de configuración que harán tu
+empezar a programar con Python.
```
```suggestion
empezar a programar con Python más fácil.
```

cmaureir commented on content/pages/es/meetups.rst:
```diff
@@ -0,0 +1,40 @@
+
+Tenemos diferentes niveles en los diferentes meetups desde principiante hasta experto en
+temas específicos. A menudo empezamos nuestros eventos con una charla de 10 minutos sobre «superpoderes
+porque creemos que la vida (tecnológica) es mucho más que programar.
```
```suggestion
Tenemos diferentes niveles en los diferentes meetups, desde principiantes hasta expertos en temas específicos. A menudo comenzamos nuestros eventos con una charla de 10 minutos sobre un "superpoder sin programar" porque creemos que la vida (tecnológica) es mucho más que programar.
```

mjmolina committed:
Apply suggestions from code review

All the suggestions seem better options than the original proposal :) !
Thank you for the review.

Co-authored-by: Cristián Maureira-Fredes <cmaureir@users.noreply.github.com>

cmaureir commented:
¡Gracias!
